### DISCUSSION TITLE
How do I integrate two different data sets for integrated analysis?

### DISCUSSION THREAD

**Discussion by pcitrano3** [satijalab/seurat#6156] [Q&A]
Hello. I am new to Seurat and I am trying to run integrated analysis, but I am having trouble following the tutorial on integrated analysis (https://satijalab.org/seurat/articles/integration_introduction.html). I have been able to complete the standard work-flow on a single data set, but I would like to try to run an integrated analysis on two different cell samples. My confusion starts at the SplitObject function line. Do I first have to merge the data sets to be able to split them? If not, how can I go about loading the two data sets and integrating them. They are independent cell lines and do not represent different conditions, as the tutorial talks about.

**Comment by klpara** [comment]
The split object line just allows the SCTransform() and FindVariableFeatures() to be done to each dataset in a merged object individually. Since your object is not merged, just run these steps then feed the objects to through the integration steps.

However, I don't see why you would not be able to merge them first, just tag each cell line with an identifier e.g., seurat@meta.data$cell_line. Your integrated object will just be one seurat object anyways in the end.
